snuba/web/delete_query.py

```diff
@@ -44,11 +47,20 @@ def delete_from_storage(
     if not delete_settings.is_enabled:
         raise Exception(f"Deletes not enabled for {storage.get_storage_key().value}")
 
-    payload: dict[str, Any] = {}
+    # assert isinstance(columns, dict[str, list[Any]])
+    columns_is_expected_type = isinstance(columns, dict) and all(
```

Comment:
I would prefer to have a schema that we validate against similar to the `RequestSchema` logic in https://github.com/getsentry/snuba/blob/e7f134852bdd33fd6d22483926463facba8ef3e0/snuba/request/schema.py#L32

I can see if I can add something in that will work for that

Reply:
The reason I added this in is bc python doesnt have runtime type checking so I needed to add the logic for it myself.

What is the reasoning you would prefer to have it like the `RequestSchema` object? Is it so that the input validation is abstracted and not a part of the function?